server/protocols/jmap-integration-testing/memory-jmap-integration-testing/src/test/java/org/apache/james/jmap/memory/MemoryGetMessageListMethodTest.java

```diff
@@ -49,4 +49,10 @@ protected void await() {
     @Test
     public void getMessageListShouldIncludeMessagesWhenTextFilterMatchesBodyWithStemming() {
     }
+
+    @Override
+    @Ignore("MemoryJmapTestRule doesn't use ElasticSearch Indexing")
```

Comment:
it does not support it, but what is failing exactly?

Reply:
> Seems like the MessageSearches used in SimpleMessageSearchIndex doesn't search on From name, but only on From Address.